### ISSUE TITLE
CLI syntax?

### ISSUE THREAD

**Issue by fivepeakwisdom** [AVGP/susi#2]
I am getting errors for everything I try.. what is the proper syntax for using susi? I really like the idea!

```
rthugsjeche@rthugsjeche ~/Projects/blog $ ls
layouts  md  public
rthugsjeche@rthugsjeche ~/Projects/blog $ ls layouts/
index.html
rthugsjeche@rthugsjeche ~/Projects/blog $ ls md/
fuzzy.md
rthugsjeche@rthugsjeche ~/Projects/blog $ ls public/
rthugsjeche@rthugsjeche ~/Projects/blog $ 
```

```
~/Projects/blog $ susi md/fuzzy.md public/ layouts/

fs.js:487
  var r = binding.read(fd, buffer, offset, length, position);
                  ^
Error: EISDIR, illegal operation on a directory
    at Object.fs.readSync (fs.js:487:19)
    at Object.fs.readFileSync (fs.js:321:28)
    at convertFilesInPath (/home/rthugsjeche/.nvm/v0.10.33/lib/node_modules/susi/susi:42:21)
    at Object.<anonymous> (/home/rthugsjeche/.nvm/v0.10.33/lib/node_modules/susi/susi:61:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
```

Also tried..

```
~/Projects/blog $ susi md public layouts

fs.js:487
  var r = binding.read(fd, buffer, offset, length, position);
                  ^
Error: EISDIR, illegal operation on a directory
    at Object.fs.readSync (fs.js:487:19)
    at Object.fs.readFileSync (fs.js:321:28)
    at convertFilesInPath (/home/rthugsjeche/.nvm/v0.10.33/lib/node_modules/susi/susi:42:21)
    at Object.<anonymous> (/home/rthugsjeche/.nvm/v0.10.33/lib/node_modules/susi/susi:61:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
```

**Comment by AVGP** [comment]
Sorry it took me so long to respond.

As it turns out the changes made Susi incompatible with relative paths..
If you run SuSi with absolute paths it's fine. That, surely, is a bug and I'll take care of it.

**Comment by AVGP** [comment]
Right, the issue is now fixed in v0.2.1

**Comment by fivepeakwisdom** [comment]
Vielen Dank!
